data/json/monstergroups/monstergroups.json

```diff
@@ -4556,7 +4556,8 @@
     "default": "mon_null",
     "monsters": [
       { "monster": "mon_zombie_nullfield", "freq": 550, "cost_multiplier": 0 },
-      { "monster": "mon_zombie_brute_shocker", "freq": 300, "cost_multiplier": 2 }
+      { "monster": "mon_zombie_brute_shocker", "freq": 300, "cost_multiplier": 2 },
+      { "monster": "mon_zombie_skeletal_electric", "freq": 300, "cost_multiplier": 2 }
```

Comment:
Either make this skeleton or the monster definition skeletal.

Reply:
Whoops. Just fixed the monster definition.